realkd/subgroups.py

```diff
@@ -9,6 +9,7 @@
 
 from realkd.search import Conjunction, Context, KeyValueProposition, Constraint, search_methods
 from realkd.rules import Rule
+from realkd.utils import validate_data
 
 
 class Impact:
```

Comment:
NOTE:
Although there is no regression in Impact, I did not end up supporting the numpy array input because 
a) I ran out of time
b) I'm trying to be super conservative with changes.